File: sms_notifications/sms.rb
require_relative 'bundle/bundler/setup'
require 'twilio-ruby'
require 'iron_worker'

sid = ENV['TWILIO_SID']                 # twilio account sid
token = ENV['TWILIO_TOKEN']             # twilio auth token

from = IronWorker.payload["from"]       # your twilio number
to = IronWorker.payload["to"]           # your phone number
body = IronWorker.payload["body"]

# set up a client to talk to the Twilio REST API
@client = Twilio::REST::Client.new sid, token

puts "Sending sms..."
r = @client.messages.create(
    :from => from,
    :to => to,
    :body => body
)

puts "Twilio response:"
p r

File: sms_notifications/php/sms.php
<?php

require 'vendor/autoload.php';
use Twilio\Rest\Client;

// get Twilio Account SID and Auth Token from env vars
$sid = getenv('TWILIO_SID');
$token = getenv('TWILIO_TOKEN');
$client = new Client($sid, $token);

// get payload (TO, FROM, BODY values)
$payload = IronWorker\Runtime::getPayload(true);

// send message
echo "Sending sms...\n";
$message = $client->messages->create($payload['to'], array(
	'From' => $payload['from'],
	'Body' => $payload['body'],
));

// get actual status of a message from Twilio
echo "Status of the message: " . $client->messages($message->sid)->fetch()->status;